_health-care/_js/components/Nav.jsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 import { Link } from 'react-router';
 
+/**
```

Comment:
First line of JSDoc should be short-sentence describing the class otherwise the tooling gets messed up.